homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,393 @@
+"""Constants for the HERE Destination Weather service."""
+from __future__ import annotations
+
+DOMAIN = "here_weather"
+
+HERE_API_KEYS = "here_api_keys"
+
+DEFAULT_SCAN_INTERVAL = 120
+
+CONF_LANGUAGE = "language"
+CONF_OFFSET = "offset"
+CONF_OPTION = "option"
+
+MODE_ASTRONOMY = "forecast_astronomy"
+MODE_HOURLY = "forecast_hourly"
+MODE_DAILY = "forecast_7days"
+MODE_DAILY_SIMPLE = "forecast_7days_simple"
+MODE_OBSERVATION = "observation"
+CONF_MODES = [
+    MODE_ASTRONOMY,
+    MODE_HOURLY,
+    MODE_DAILY,
+    MODE_DAILY_SIMPLE,
+    MODE_OBSERVATION,
+]
+DEFAULT_MODE = MODE_DAILY_SIMPLE
+
+ASTRONOMY_ATTRIBUTES: dict[str, dict[str, str | None]] = {
+    "sunrise": {"name": "Sunrise", "unit_of_measurement": None},
+    "sunset": {"name": "Sunset", "unit_of_measurement": None},
+    "moonrise": {"name": "Moonrise", "unit_of_measurement": None},
+    "moonset": {"name": "Moonset", "unit_of_measurement": None},
+    "moonPhase": {"name": "Moon Phase", "unit_of_measurement": "%"},
+    "moonPhaseDesc": {"name": "Moon Phase Description", "unit_of_measurement": None},
+    "city": {"name": "City", "unit_of_measurement": None},
+    "latitude": {"name": "Latitude", "unit_of_measurement": None},
+    "longitude": {"name": "Longitude", "unit_of_measurement": None},
+    "utcTime": {"name": "UTC Time", "unit_of_measurement": None},
```

Comment:
UTC time could be a sensor with device class timestamp?

Reply:
Done in https://github.com/home-assistant/core/pull/28910/commits/40949577d7e7c78c0ec625ff004b5e0d14db179f